### PR TITLE
[12.0][sql_export_mail] Migration from v9

### PR DESCRIPTION
**Pull request by florian-dacosta**
Allow to send some file generated by sql_export by mail regularly (by cron).

@aheficent @bealdav
I think you use sql_export, you may be interested.

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
It can be useful and it works. :+1:

File: sql_export_mail/models/sql_export.py [hunk 1]
```diff
@@ -0,0 +1,144 @@
+# Copyright 2019 Akretion
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api, _
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+from odoo.exceptions import Warning as UserError
```
Comment (bealdav) [inline sql_export_mail/models/sql_export.py]: no more Warning

File: sql_export_mail/models/sql_export.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        """
+            Called from mail template
+        """
+        self.ensure_one()
+        if self.env.context.get('mail_to'):
+            mail_users = self.env['res.users'].browse(
+                self.env.context.get('mail_to'))
+        else:
+            mail_users = self.mail_user_ids
+        emails = ''
```
Comment (bealdav) [inline sql_export_mail/models/sql_export.py]:
```suggestion
        return ','.join([x.email for x in mail_users])
```

not really tested but something like that

Comment (bealdav) [reply sql_export_mail/models/sql_export.py]: you can remove the rest

**Review by bealdav — APPROVED** [review]
Thanks a lot.
Nice ok for me except suggestion

**Comment by florian-dacosta** [comment]
@bealdav Done, thanks

**Comment by bealdav** [comment]
nice

**Comment by florian-dacosta** [comment]
@legalsylvain And this one is also ready to merge, could you do it?
Thanks!

**Pull request merged**